### PR TITLE
Create test with rspec

### PR DESCRIPTION
**Pull request by silvitaelen**
- Installation of gems: database_cleaner, faker and factory_bot.
- Creating tests for Book model with Rspec.

TRELLO CARD: https://trello.com/c/unhJJLKg

### CODE REVIEW

File: spec/models/book_spec.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+describe Book, type: :model do
+  it { should validate_presence_of(:genre) }
+  it { should validate_presence_of(:author) }
+  it { should validate_presence_of(:image) }
+  it { should validate_presence_of(:title) }
+  it { should validate_presence_of(:publisher) }
+  it { should validate_presence_of(:year) }
+
+  subject(:book) { create(:book) }
```
Comment (Atixx) [inline spec/models/book_spec.rb]: Move this to the top, so the spec's subject is clearer

File: spec/models/book_spec.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+  it { is_expected.to be_valid }
+
+  describe '#generate_verification_code' do
```
Comment (Atixx) [inline spec/models/book_spec.rb]: which behavior do you want to test here? the book model doesn't generate codes

**Review by Atixx — CHANGES_REQUESTED** [review]

**Comment by Atixx** [comment]
Also, please make sure to follow the pull request template for formatting

File: spec/models/book_spec.rb (hunk 2 above)
Comment (silvitaelen) [reply spec/models/book_spec.rb]: I used the example of card and forgot to eliminate what I would not use

**Review by Atixx — APPROVED** [review]
👍

**Pull request merged**